src/YoutubeAtom.tsx

```diff
@@ -14,6 +14,7 @@ import type { ArticleTheme } from '@guardian/libs';
 import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
 
 type Props = {
+    id: string;
     assetId: string;
```

Comment:
Is it worth refactoring things a bit to reduce the number of variable names we have relating to ids? Is `assetId` always the same as `videoId`? And is `id` always `elementId`?

Reply:
I agree this could be better!

I think there are two questions here:

- do we require both ids?
- can we rename the props to be more meaningful?

#### Required ids

We do need both ids:

- `assetId` (locally `videoId`) for the YouTube player
- `id` for uniqueness

#### Naming

I wasn't sure what to do with the naming as the conventions come from DCR.

`id` (aka `element.id`) is the prop we get from DCR. `elementId` is more descriptive but `id` follows the [existing naming convention](https://github.com/guardian/dotcom-rendering/blob/a78a98b3001cc73be46f52078bd75153d53741a2/dotcom-rendering/src/web/lib/renderElement.tsx#L458-L512).

`assetId` is the prop we get from DCR but it is locally renamed  to `videoId` to be more meaningful in the local context.

What I can do is push up the renamed props to the `YoutubeBlockComponent` level so within `YoutubeAtom` we have consistent naming?

|DCR| YoutubeAtom naming | Proposed YoutubeBlockComponent naming |
|:---|:---|:---|
| assetId | videoId | videoId |
| id | id | elementId |